apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -661,3 +673,23 @@ listener_identifier(Protocol, ListenOn) ->
         ID ->
             ID
     end.
+
+restart_emqx_management(Identifier) ->
```

Comment:
I noticed that restart_emqx_management/1 and restart_emqx_dashboard/1 are almost in the same structure. Then rewrite it into a common function would make sense.